All log types enabled/delivered by default

paullizer opened the pull request:
## Describe your Changes
All logs are now attempted to collect and deliver to CISA CLAW, if the agency is not yet collecting them, then the scheduled task has nothing to deliver but when the agency does send other logs then they will automatically start being uploaded to the CLAW. I wanted to eliminate the additional overhead of having the agency dig into the automated task and enable log delivery.

Additional improvement to determine if logs are sent to the Log Analytics workspace, if the log table is absent, then the task exits gracefully. If the table exists, then the last 15 minutes of logs are collected and delivered to the CLAW.

## Issue ticket number and link if relevant

## Checklist before requesting a review
- [X] I have performed a self-review of my code and documentation
- [X] I have updated the README to denote my solution/training/whitepaper
- [X] I have identified which clouds this can run (Azure Cloud / Azure Government)

haithamshahin333 commented on topics/infrastructure/solutions/tic3.0/Azure-Application-Gateway/azuredeploy.json:
```diff
@@ -58,9 +38,8 @@
     },
     "variables": {
-        "webAppDnsName": ".azurewebsites",
+        "webAppDnsName": ".azurewebsites.net",
```
@paullizer  is this intentional? or do we want to keep the var for being environment-agnostic.

paullizer replied:
Updated domain names for web service and front door to support both MAC and MAG. Also updated job schedule names, automation account, and runbook to be unique within tenant.

paullizer commented:
Updated domain names for web service and front door to support both MAC and MAG. Also updated job schedule names, automation account, and runbook to be unique within tenant.
